src/clients/InventoryClient.ts

```diff
@@ -459,6 +462,7 @@ export class InventoryClient {
     // since its the fallback chain if both destination and origin chain are over allocated.
     // If destination chain is hub chain, we still want to evaluate it before the origin chain.
     if (
+      !deposit.fromLiteChain &&
       !chainsToEvaluate.includes(destinationChainId) &&
       this._l1TokenEnabledForChain(l1Token, Number(destinationChainId))
```

Comment:
Minor: The majority of deposits will not originate from a lite chain so I think it makes sense to evaluate this last.

```suggestion
      !chainsToEvaluate.includes(destinationChainId) &&
      this._l1TokenEnabledForChain(l1Token, Number(destinationChainId)) &&
      !deposit.fromLiteChain
```